function handleError(err, res) {
  console.log(err.message);
}

module.exports = { handleError };